internal/action/init.go

```diff
@@ -55,6 +55,15 @@ func (s *Action) IsInitialized(c *cli.Context) error {
 	out.Printf(ctx, logo)
 	out.Printf(ctx, "🌟 Welcome to gopass!")
 	out.Noticef(ctx, "No existing configuration found.")
+
+	contSetup, err := termio.AskForBool(ctx, "❓ Do you want to continue to setup?", true)
```

Comment:
The third parameter to AskForBool is the default. I think that should be false in this case.

If gopass is run non-interactively it will always use the default value and we only want to start the setup flow if the user explicitly asks for it. So please change `true` to `false` and then we should be good.